cmd/view-fits/main.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2015 The astrogo Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"image"
+	"image/color"
+	"image/draw"
+	"log"
+	"os"
+
+	"github.com/astrogo/fitsio"
+
+	"golang.org/x/exp/shiny/driver"
+	"golang.org/x/exp/shiny/screen"
+	"golang.org/x/mobile/event/key"
+	"golang.org/x/mobile/event/paint"
+)
+
+var (
+	hduArray []fitsio.Image
```

Comment:
come to think about it some more...
what about going the other directions:

- create a `fileInfo` struct such as:
```go
type fileInfo struct {
    Name string
    Images []image.Image
}
```

- use Left/Right arrows to navigate through `fileInfo`s 
- use Up/Down arrows to navigate through a `fileInfo`'s images
- replace `hduArray []fitsio.Image` with `infos []fileInfo`

what do you think ?